src/lib/utils.js

```diff
@@ -4,9 +4,16 @@ export const makeLog = context => (...args) =>
   // eslint-disable-next-line no-console
   DEBUG && console.log(`[ThemesRFun ${context}]`, ...args);
 
+export const floorHSLA = color => ({
```

Comment:
Planning to revisit this in #50 - the color picker and theme schema use different magnitudes for HSLA values, but the JS floating point math causes the values to drift. And thus, themes that should be identical look non-identical to the shouldOfferPendingTheme selector because insignificant digits have changed.